### PR TITLE
Catalog tests

### PR DESCRIPTION
**Pull request by VeereshAradhya**
Catalog tests

### CODE REVIEW

File: testdata/v1beta1/clustertask/pipelines/jib-maven-pz.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      workspace: source
+    params:
+    - name: DIRECTORY
+      value: $(params.SUBDIR)
+    - name: IMAGE
+      value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/jib-maven
+    - name: INSECUREREGISTRY
+      value: "false"
+    - name: MAVEN_IMAGE
+      value: maven:3.6.3-adoptopenjdk-11
```
Comment (ppitonak) [inline testdata/v1beta1/clustertask/pipelines/jib-maven-pz.yaml:44]: let's update it to `maven:3.8.3-adoptopenjdk-11` - it worked fine when I was testing 1.8.0

File: testdata/v1beta1/pipelinerun/buildah-disconnected-pipelinerun.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+metadata:
+  name: shared-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: tekton.dev/v1beta1
```
Comment (ppitonak) [inline testdata/v1beta1/pipelinerun/buildah-disconnected-pipelinerun.yaml:80]: Why do we have this directory? These pipelines, pvc and pipelineruns are already defined in `clustertask` subdir

File: testdata/v1beta1/clustertask/pipelineruns/skopeo-copy.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+spec:
+  pipelineSpec:
+    tasks:
+    - name: run-skopeo-copy
+      taskRef:
+        name: skopeo-copy
+        kind: ClusterTask
+      params:
+      - name: srcImageURL
+        value: "docker://quay.io/fedora/fedora:34"
```
Comment (ppitonak) [inline testdata/v1beta1/clustertask/pipelineruns/skopeo-copy.yaml:14]: let's update to 36 in both srcImageURL and destImageURL

**Review by ppitonak — CHANGES_REQUESTED** [review]
s2i tests will be handled in new PR?

**Comment by VeereshAradhya** [comment]
Yes, S2I tests will be added in a separate PR

File: testdata/v1beta1/pipelinerun/buildah-disconnected-pipelinerun.yaml (hunk 2 above)
Comment (VeereshAradhya) [reply testdata/v1beta1/pipelinerun/buildah-disconnected-pipelinerun.yaml:80]: This directory is for pipelineruns.

**Review by ppitonak — APPROVED** [review]

**Pull request merged**